src/sync/sync_permission.hpp

```diff
@@ -93,12 +93,24 @@ class PermissionResults {
         return m_results.async(std::move(target));
     }
 
-    // Create a new instance by further filtering or sorting this instance.
+    // Create a new instance by further filtering this instance.
     PermissionResults filter(Query&& q) const
     {
         return PermissionResults(m_results.filter(std::move(q)));
     }
 
+    // Create a new instance by sorting this instance.
+    PermissionResults sort(SortDescriptor&& s) const
+    {
+        return PermissionResults(m_results.sort(std::move(s)));
+    }
+
+    // Get the results.
+    Results& results()
```

Comment:
I use it a few times in [this commit](https://github.com/realm/realm-cocoa/pull/4799/commits/4543d5365edd4e509c5d01ac4fd1ea52b03f01d2) (search for `_results->results()`). Much of the Cocoa machinery to assemble sort descriptors and other thing assumes you have easy access to the raw results value.